app/package.json

```diff
@@ -150,5 +150,8 @@
 		"vue-tsc": "2.0.29",
 		"vuedraggable": "4.1.0",
 		"wellknown": "0.5.0"
+	},
+	"dependencies": {
+		"string-width": "7.2.0"
```

Comment:
Lets move this to `devDependencies`.

The reason is that we bundle the app before it's published to `npm`, so when you `npm install @directus/app` you already get the full bundled dist, so it shouldn't re-download the dependencies as well 🙂

Reply:
Got it. 👍